content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -372,6 +374,18 @@ implemented using the NetworkPolicy API.
 - The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost
   access, nor do they have the ability to block access from their resident node).
 
+## Recommendations for Users
+
+This section covers a set of recommended best practices to follow when using NetworkPolicies.
+
+- NetworkPolicies apply to connections and not packets. Thus if a NetworkPolicy is applied
+  in the middle of an existing connection its implementation defined as to whether the policy
```

Comment:
nit, but should be "it's" with an apostrophe as a contraction for "it is"

Reply:
done